client/src/Pages/About.js

```diff
@@ -0,0 +1,17 @@
+
+const About = () => {
+return (
+	<div
+	style={{
+		display: 'flex',
+		justifyContent: 'Right',
+		alignItems: 'Right',
+		height: '100vh'
+	}}
```

Comment:
Given the repetition, you might want to consider using classes or a wrapping element for these styles.

Reply:
K will do